akka-http2-support/src/main/scala/akka/http/impl/engine/http2/client/RequestRendering.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.impl.engine.http2
+package client
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import akka.annotation.InternalApi
+import akka.event.LoggingAdapter
+import akka.http.impl.engine.http2.FrameEvent.ParsedHeadersFrame
+import akka.http.scaladsl.model.http2.RequestResponseAssociation
+import akka.http.scaladsl.model.{ ContentTypes, HttpEntity, HttpRequest }
+
+import scala.collection.immutable.VectorBuilder
+
+@InternalApi
+private[http2] object RequestRendering {
```

Comment:
For parsing and rendering, I adopted the style of the server-side components. In the future, more of those components could be shared.